Misc/NEWS.d/next/Library/2024-10-11-04-04-38.gh-issue-125260.PeZ0Mb.rst

```diff
@@ -0,0 +1 @@
+The :func:`gzip.compress` *mtime* parameter now defaults to 0 for reproducible output.
```

Comment:
Add credits for yourself (feel free to remove me if you'd prefer, I only helped with tests/docs)
```suggestion
The :func:`gzip.compress` *mtime* parameter now defaults to 0 for reproducible output.
Patch by Bernhard M. Wiedemann and Adam Turner.
```